### PR TITLE
feat: header, sidenav, profile menu

### PR DESCRIPTION
**Pull request by Anastasiia-Svechnikova**
Feat: Header, sidenav, user menu

### CODE REVIEW

File: src/app/navigation/components/header/header.component.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+
+@Component({
```
Comment (AleaJactaEst) [inline src/app/navigation/components/header/header.component.ts:3]: let's by the way add rule to always use: ChangeDetectionStrategy - OnPush

File: src/app/navigation/components/site-layout/site-layout.component.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  constructor(
+    private route: ActivatedRoute,
+    private cdref: ChangeDetectorRef,
+  ) {}
+
+  setHeader(): void {
+    this.title = this.route.snapshot.firstChild?.data['header'];
+  }
+  ngAfterViewInit(): void {
+    this.cdref.detectChanges();
```
Comment (AleaJactaEst) [inline src/app/navigation/components/site-layout/site-layout.component.ts:28]: Looks like some hack solution, please avoid this

File: src/app/navigation/constants/constants.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import { UserRoles } from 'src/app/shared/constants/constants';
+
+export const sideNavigationLinksData = [
+  {
+    path: '/home',
+    iconName: 'home',
+    title: 'home',
+    accessRoles: [UserRoles.Admin, UserRoles.Guest],
```
Comment (AleaJactaEst) [inline src/app/navigation/constants/constants.ts:8]: Is those UserRoles same as on BE side?

File: src/app/navigation/components/site-layout/site-layout.component.scss
Comment (Las04ka) [inline src/app/navigation/components/site-layout/site-layout.component.scss:1]: Let’s add new line between each style block

Comment (Las04ka) [inline src/app/navigation/components/site-layout/site-layout.component.scss:1]: Let’s add new line between each style block

**Review by Las04ka — APPROVED** [review]

**Pull request merged**